Index: llm/.idea/.gitignore
===================================================================
diff --git a/llm/.idea/.gitignore b/llm/.idea/.gitignore
deleted file mode 100644
--- a/llm/.idea/.gitignore	(revision b0d9378b0307bc23317b23ae4f962f904f0f102f)
+++ /dev/null	(revision b0d9378b0307bc23317b23ae4f962f904f0f102f)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
